### PR TITLE
Update readme http links to https

### LINKED ISSUE #9243: Update readme http links to https
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [README.md](https://github.com/publiclab/plots2/blob/main/README.md) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -3,7 +3,7 @@ PublicLab.org
 
 [![Code of Conduct](https://img.shields.io/badge/code-of%20conduct-green.svg)](https://publiclab.org/conduct)
 [![Build Status](https://github.com/publiclab/plots2/workflows/tests/badge.svg?branch=main)](https://github.com/publiclab/plots2/actions)
-[![first-timers-only-friendly](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
+[![first-timers-only-friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
 [![Join the chat at https://publiclab.org/chat](https://img.shields.io/badge/chat-in%20different%20ways-blue.svg)](https://publiclab.org/chat)
 [![Code Climate](https://codeclimate.com/github/publiclab/plots2/badges/gpa.svg)](https://codeclimate.com/github/publiclab/plots2)
 [![codecov](https://codecov.io/gh/publiclab/plots2/branch/main/graph/badge.svg)](https://codecov.io/gh/publiclab/plots2)
@@ -23,11 +23,11 @@ Some key features include:
 * A [Q&A portal](https://publiclab.org/questions) for peer-based problem solving
 * A rich text and Markdown [editor](https://github.com/publiclab/PublicLab.Editor)
 * [Wiki editing](https://publiclab.org/wiki) and revision tracking
-* Tagging and [topically-based groups and content organization](http://publiclab.org/tags)
+* Tagging and [topically-based groups and content organization](https://publiclab.org/tags)
 * Email notification subscriptions for topics and comments
 * A search interface built out of [our growing API](https://github.com/publiclab/plots2/blob/main/doc/API.md)
 * A user dashboard [presenting recent activity](https://publiclab.org/dashboard)
-* A privacy-sensitive, Leaflet-based [location tagging system](https://github.com/publiclab/leaflet-blurred-location/) and [community map](http://publiclab.org/people)
+* A privacy-sensitive, Leaflet-based [location tagging system](https://github.com/publiclab/leaflet-blurred-location/) and [community map](https://publiclab.org/people)
 
 ## Roadmap
 
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
2. If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

3. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by fannyibz**
Fixes #9243(<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by fannyibz** [comment]
Hi, 
I did my first pull request. I don't understand why "tests / unit-tests (pull_request) Failing after 1m — unit-tests" is failing.
Could you help me please?
Thanks a lot

**Comment by cesswairimu** [comment]
Hey @fannyibz , we have one test fails sometimes on this...I will close this and reopen to if it passes...In the meantime, kindly add the issue link to the issue this pull request fixes on your pull request on this line "Fixes issue_link" this will help other contributors while reviewing your changes. It also helps in automatically closing the issue when the pull request is merged. Thanks

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ PublicLab.org
 
 [![Code of Conduct](https://img.shields.io/badge/code-of%20conduct-green.svg)](https://publiclab.org/conduct)
 [![Build Status](https://github.com/publiclab/plots2/workflows/tests/badge.svg?branch=main)](https://github.com/publiclab/plots2/actions)
-[![first-timers-only-friendly](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
++[![first-timers-only-friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
```
Comment (cesswairimu) [inline README.md:6]: @fannyibz kindly remove the `+`  at the beginning of the lines you changes...on the issue description the `+` represented the lines that were to be added  and the lines starting with `-` were to be remove...sorry for the confusion.

File: README.md [hunk 2]
```diff
@@ -23,11 +23,12 @@ Some key features include:
 * A [Q&A portal](https://publiclab.org/questions) for peer-based problem solving
 * A rich text and Markdown [editor](https://github.com/publiclab/PublicLab.Editor)
 * [Wiki editing](https://publiclab.org/wiki) and revision tracking
-* Tagging and [topically-based groups and content organization](http://publiclab.org/tags)
++* Tagging and [topically-based groups and content organization](https://publiclab.org/tags)
```
Comment (cesswairimu) [inline README.md:26]: remove the `+` here

File: README.md [hunk 3]
```diff
@@ -23,11 +23,12 @@ Some key features include:
...
 * Email notification subscriptions for topics and comments
 * A search interface built out of [our growing API](https://github.com/publiclab/plots2/blob/main/doc/API.md)
 * A user dashboard [presenting recent activity](https://publiclab.org/dashboard)
-* A privacy-sensitive, Leaflet-based [location tagging system](https://github.com/publiclab/leaflet-blurred-location/) and [community map](http://publiclab.org/people)
++* A privacy-sensitive, Leaflet-based [location tagging system](https://github.com/publiclab/leaflet-blurred-location/) and [community map](https://publiclab.org/people)
```
Comment (cesswairimu) [inline README.md:30]: remove the `+` here too

**Review by cesswairimu — CHANGES_REQUESTED** [review]
Everything else looks good and we should be good to merge after the updates...Thanks for working on this

**Review by cesswairimu — APPROVED** [review]
Great :tada: thanks for working on this

**Pull request merged**